### PR TITLE
[Netmanager] Fix network uptime graph time & update battery voltage chart

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)

- [x] Fixed network uptime to reflect most recent time
- [x] Refactored apex chart component to use a function component 
- [x] Update voltage chart with new api
- [x] Updated apex chart library
- [x] Added minutes average filter to battery voltage chart

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [AN-485](https://airqoteam.atlassian.net/browse/AN-485) (If exists)

#### Screenshots (optional)
image
image
image

[AN-485]: https://airqoteam.atlassian.net/browse/AN-485?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by 123MwanjeMike** [comment]
Great work @Codebmk

**Review by NoahNsimbe — COMMENTED** [review]
Thanks, @Codebmk. Could we set the default to raw (not averaged) and add lower averages such as 5, 10 and 15 minutes?

**Comment by Codebmk** [comment]
> Thanks, @Codebmk. Could we set the default to raw (not averaged) and add lower averages such as 5, 10 and 15 minutes?

What do you mean? @NoahNsimbe  Should this be a parameter for the battery voltage api?

**Comment by NoahNsimbe** [comment]
> > Thanks, @Codebmk. Could we set the default to raw (not averaged) and add lower averages such as 5, 10 and 15 minutes?
> 
> What do you mean? @NoahNsimbe Should this be a parameter for the battery voltage api?
Screenshot 2023-07-13 at 17 18 05

**Comment by Codebmk** [comment]
> > > Thanks, @Codebmk. Could we set the default to raw (not averaged) and add lower averages such as 5, 10 and 15 minutes?
> > 
> > 
> > What do you mean? @NoahNsimbe Should this be a parameter for the battery voltage api?
> > Screenshot 2023-07-13 at 17 18 05


I have resolved that. Chart displays raw data by default and the user can choose to average the data
image

**Review by NoahNsimbe — APPROVED** [review]
LGTM. Thanks @Codebmk

**Review by Baalmart — COMMENTED** [review]
thanks @Codebmk , just have a small problem which is shown in the attached screenshot.
Screenshot 2023-07-13 at 23 11 30

**Review by Baalmart — COMMENTED** [review]
thanks @Codebmk , a few more feedback comments. Using screenshots... :) 
Screenshot 2023-07-13 at 23 33 59
Screenshot 2023-07-13 at 23 36 08

**Comment by Codebmk** [comment]
@Baalmart the battery voltage data is displayed raw by default(ie not averaged). When you set a minute average value, then it is averaged by the selected minutes

**Comment by Baalmart** [comment]
> @Baalmart the battery voltage data is displayed raw by default(ie not averaged). When you set a minute average value, then it is averaged by the selected minutes

Thanks @Codebmk how does one go (navigate) back to RAW after selecting any of the averaged values?

**Comment by Codebmk** [comment]
@Baalmart in the deploy preview the time displayed in uniform...
image


> > @Baalmart the battery voltage data is displayed raw by default(ie not averaged). When you set a minute average value, then it is averaged by the selected minutes
> 
> Thanks @Codebmk how does one go (navigate) back to RAW after selecting any of the averaged values?

Oh okay, I have resolved that. Deploy preview loading..

File: netmanager/src/views/charts/utils.js [hunk 1]
```diff
@@ -1,51 +1,63 @@
...
+      showDuplicates: false
+    }
   },
   tooltip: {
     x: {
-      format: "dd MMM yyyy hh:mm:ss",
-    },
+      formatter: function (val) {
+        return moment(new Date(val)).format('DD MMM yyyy HH:mm:ss');
+      }
```
Comment (Codebmk) [inline netmanager/src/views/charts/utils.js:31]: Use 24 hour time in tooltip

File: netmanager/src/views/components/DataDisplay/DeviceView/DeviceOverview/DeviceOverviewChart.js [hunk 2]
```diff
@@ -1,46 +1,100 @@
...
+              <option value={'5'}>5</option>
+              <option value={'20'}>10</option>
+              <option value={'30'}>20</option>
+              <option value={'30'}>30</option>
+              <option value={'45'}>40</option>
+              <option value={'60'}>60</option>
+              <option value={'90'}>80</option>
+              <option value={'90'}>100</option>
+              <option value={'120'}>120</option>
+            </TextField>
```
Comment (Codebmk) [inline netmanager/src/views/components/DataDisplay/DeviceView/DeviceOverview/DeviceOverviewChart.js:86]: Averaging and raw data selections for battery voltage chart

File: netmanager/src/views/charts/utils.js (hunk 1 above)
Comment (Baalmart) [reply netmanager/src/views/charts/utils.js:31]: thanks @Codebmk

File: netmanager/src/views/components/DataDisplay/DeviceView/DeviceOverview/DeviceOverviewChart.js (hunk 2 above)
Comment (Baalmart) [reply netmanager/src/views/components/DataDisplay/DeviceView/DeviceOverview/DeviceOverviewChart.js:86]: thanks!

**Pull request merged**